### ISSUE TITLE
Status of this project?

### ISSUE THREAD

**Issue by dputtick** [nodejs/repl#46]
:wave: Hi @devsnek. I'm curious what the status of this project is – I've been poking around improving the existing node REPL a bit, and came across this repo. Is there a plan to eventually move it into mainline node? If yes, do you have an idea of the features that are missing/work that remains to be done in your opinion before heading in that direction? And, how much latitude do you feel there is to break the existing REPLServer interface?

**Comment by devsnek** [comment]
@dputtick this will never replace `require('repl')`, but there is a small chance it will replace node's interactive mode. I don't have any timeline or concrete plans to do so though. At the moment, just getting the features running to experiment with and then adding them to the upstream repl has been working for us.

**Comment by dputtick** [comment]
@devsnek ah, ok. Do the folks working on node repl + readline stuff have a mailing list/irc channel things tend to get discussed in, or does that usually happen in tickets in https://github.com/nodejs/node? Also, would you be ok with me PRing the gist of your comment above to the readme to make the purpose of this repo more clear to passers-by?

**Comment by devsnek** [comment]
@dputtick #node-dev on irc.freenode.net and github issues are the main communication channels we have.

**Comment by guybedford** [comment]
@dputtick were you able to answer all your questions on this? Are you still interested in getting involved with Node.js REPL work?

**Comment by dputtick** [comment]
@guybedford I did get them answered, but working on the existing repl code without feeling comfortable making breaking changes started to feel like kind of a slog. I think I decided it might be a better project for somebody who's already an established node contributor. I am still somewhat interested, but I write more Ruby than js at work at the moment, so I'm not as aware of my pain points as I was when I made this issue.
